.github/workflows/esti.yaml

```diff
@@ -956,6 +956,7 @@ jobs:
           LAKEFS_BLOCKSTORE_TYPE: azure
           ESTI_BLOCKSTORE_TYPE: azure
           ESTI_STORAGE_NAMESPACE: https://esti4hns.blob.core.windows.net/esti-system-testing/${{ github.run_number }}/${{ steps.unique.outputs.value }}
+          ESTI_LARGE_OBJECT_PATH: https://esti4hns.blob.core.windows.net/esti-system-testing-data/copy-test-data/data.6mib
```

Comment:
```suggestion
          ESTI_AZURE_LARGE_OBJECT_PATH: https://esti4hns.blob.core.windows.net/esti-system-testing-data/copy-test-data/data.6mib
```

Reply:
Not done. This would be possible on _any_ block adapter. We just chose to use it on Azure thingy genV2. So naming it after Azure puts usage into operation.